tls/s2n_client_key_exchange.c

```diff
@@ -30,7 +30,9 @@
 #include "utils/s2n_safety.h"
 #include "utils/s2n_random.h"
 
-static int s2n_rsa_client_key_recv(struct s2n_connection *conn)
+static int calculate_keys(struct s2n_connection *conn, struct s2n_blob *shared_key);
```

Comment:
Style: I don't like such forward declarations because they appear twice on searches. In addition, when you want to change the interface you will need to change it twice. I prefer instead to put the function implementation here and avoid the declaration.